lib/src/cupertino/cupertino_toolbar.dart

```diff
@@ -79,6 +83,14 @@ class CupertinoPopoverToolbar extends StatefulWidget {
   /// All of the items will have this exact height.
   final double height;
 
+  /// The z-coordinate relative to the parent at which to place this toolbar.
+  ///
+  /// The value is non-negative.
+  final double elevation;
+
+  /// The shadow color.
```

Comment:
Please include info about how the alpha is used. Will a shadow be rendered completely opaque black if the user passes black?

Reply:
The shadow opacity is controlled by the `elevation`. I added this to the docs.